plugins/woocommerce-blocks/assets/js/editor-components/sidebar-compatibility-notice/index.tsx

```diff
@@ -11,17 +11,26 @@ import classnames from 'classnames';
  */
 import './editor.scss';
 import { useCompatibilityNotice } from './use-compatibility-notice';
+import { SwitchToClassicShortcodeButton } from '../switch-to-classic-shortcode-button';
+
+interface CartCheckoutSidebarCompatibilityNoticeProps {
+	block: 'cart' | 'checkout';
+	clientId: string;
+}
 
 export const CartCheckoutSidebarCompatibilityNotice = ( {
 	block,
-}: {
-	block: 'cart' | 'checkout';
-} ) => {
+	clientId,
+}: CartCheckoutSidebarCompatibilityNoticeProps ) => {
 	const [ isVisible, dismissNotice ] = useCompatibilityNotice( block );
 
+	if ( ! isVisible ) {
+		return null;
+	}
+
 	const noticeText = createInterpolateElement(
 		__(
-			'The Cart & Checkout Blocks are built to optimize for faster checkout. To make sure this feature is right for your store, <a>review the list of compatible extensions</a>.',
+			"Some extensions don't yet support this block, which may impact the shopper experience. <a>Learn more</a>",
```

Comment:
I'm sorry Niels 😅, are we supposed to drop the "review the list yada yada" part? What we got now mismatches the issue info

<img width="319" alt="image" src="https://github.com/woocommerce/woocommerce/assets/17236129/b01ac42f-74cb-4e0a-9a39-1d7eca6bd35d">

Reply:
Thanks for catching that, @wavvves. You're correct. According to the acceptance criteria in #42072, the current message was incorrect. I just corrected that. You can now review this PR again.